src/widgets/FileManagerWidget.php

```diff
@@ -30,18 +32,118 @@ class FileManagerWidget extends BaseFileManagerWidget
     /**
      * @var string
      */
-    public $title = 'My Filemanager';
+    public $title = '';
 
     /**
      * @var string
      */
     public $template = "<div data-ng-app=\"FileManagerApp\"><div class=\"ng-cloak\"><angular-filemanager></angular-filemanager></div></div>";
 
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        // register assets
+        AfmAsset::register(\Yii::$app->view);
+
+        // Config filemanager
+        $this->setFileManagerConfig();
+    }
+
     /**
      * @return string
      */
     public function run()
     {
         return $this->template;
     }
+
+    /**
+     * Set handler url and init angular module config
+     */
+    protected function setFileManagerConfig()
+    {
+        // Set handler Url
+        if ($this->handlerUrl === null) {
+            $this->handlerUrl = Url::to([getenv('AFM_HANDLER_URL')]);
+        }
+
+        // check if user is allowed to change permissions
+        $allowPermissions = false;
+        if (!\Yii::$app->user->isGuest && \Yii::$app->user->can('FileflyPermissions')) {
+            $allowPermissions = true;
+        }
+
+        $title             = empty($this->title) ? getenv('AFM_TITLE') : $this->title;
```

Comment:
This line is also changed compared to the original version, something else we need to look at?

Reply:
Yes you‘re right schmunk I better should have splitted the MR into two commits... scusi!

No, besides merging the base class and init the title property with an empty dtring its the only line.